typings/index.d.ts

```diff
@@ -10,4 +10,5 @@
 declare module 'axios/lib/adapters/xhr';
 
 declare module 'find-cypress-specs';
+declare module 'dom-to-image-more';
```

Comment:
@elastic/kibana-core please let me know if this is ok here. I added it because of this error:
```
 error TS7016: Could not find a declaration file for module 'dom-to-image-more'. '/Users/stephmilovic/dev/kibana/node_modules/dom-to-image-more/dist/dom-to-image-more.min.js' implicitly has an 'any' type.
 proc [tsc]   Try `npm i --save-dev @types/dom-to-image-more` if it exists or add a new declaration (.d.ts) file containing `declare module 'dom-to-image-more';`
 proc [tsc]
 proc [tsc] 10 import domtoimage from 'dom-to-image-more';
 proc [tsc]                           ~~~~~~~~~~~~~~~~~~~
 ```